src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -90,8 +90,7 @@ public function handleFindologicSearchParams(Request $request): ?string
         $params = array_merge($queryParams, $mappedParams);
 
         return sprintf(
```

Comment:
With this change the `sprintf` is basically pretty useless. You could just return `'?' . http_build_query(...)`.